Making event-bus examples CE spec 3.0 compliant (#129)
kyma-project/examples: event-subscription/lambda/README.md

        ```bash
        kubectl logs -f $(kubectl get po -n $KYMA_EXAMPLE_NS -l function=hello-with-data --no-headers | grep -i running | awk '{print $1}') -c hello-with-data -n $KYMA_EXAMPLE_NS
        ```

> 6. To access attributes of events which are compliant to CloudEvents [specification](https://github.com/cloudevents/spec/blob/v0.3/http-transport-binding.md), you can access event object as follows:

Reviewer: ```suggestion
6. To access Event attributes compliant with the CloudEvents [specification](https://github.com/cloudevents/spec/blob/v0.3/http-transport-binding.md) use the following paths in the request:
```
Question: can we write "paths" here? How are the attributes accessed - is it a specific request which uses the path?

Author: Yes we can and with more context.
`use the following paths for the event object in the request:`
Will it make sense?
